src/compute/src/server.rs

```diff
@@ -227,10 +228,11 @@ pub async fn compute_node_serve(
                 sstable_object_id_manager: storage.sstable_object_id_manager().clone(),
                 task_progress_manager: Default::default(),
                 await_tree_reg: None,
+                running_task_count: Arc::new(AtomicU32::new(0)),
             });
 
             let (handle, shutdown_sender) =
-                Compactor::start_compactor(compactor_context, hummock_meta_client);
+                Compactor::start_compactor(compactor_context, hummock_meta_client, 2.0);
```

Comment:
if we start compute with embedded_compactor_enabled , it only use "1" worker_num, so "2.0" is always enought although its not important.

 And  when compactor is started separately, it will be configured with a separate opt